File: Backend/QuickMart/src/main/java/com/garyMeledath/QuickMart/Dto/Purchase.java
package com.garyMeledath.QuickMart.Dto;

import java.util.Set;

import com.garyMeledath.QuickMart.Entity.Address;
import com.garyMeledath.QuickMart.Entity.Customer;
import com.garyMeledath.QuickMart.Entity.Order;
import com.garyMeledath.QuickMart.Entity.OrderItem;

import lombok.*;

@Data
public class Purchase {
	

	private Customer customer;
	private Address shippingAddress;
	private Address billingAddress;
	private Order order;
	private Set<OrderItem> orderItems;
	
}

File: QuickMart_Frontend/src/app/common/purchase.ts
import { Address } from "./address";
import { Customer } from "./customer";
import { Order } from "./order";
import { OrderItem } from "./order-item";

export class Purchase {

    public customer!: Customer;
    public shippingAddress!: Address;
    public billingAddress!:Address;
    public order!:Order;
    public orderItems!: OrderItem[]
  purchase: any;


    constructor(){}
    
}
